### PR TITLE
refactor(storage): refactor get_from_table

### PR DESCRIPTION
**Pull request by Li0k**
I hereby agree to the terms of the [Singularity Data, Inc. Contributor License Agreement](https://gist.github.com/skyzh/0663682a70b0edde7ae991492f2314cb#file-s9y_cla).

refactor get_from_table

***PLEASE DO NOT LEAVE THIS EMPTY !!!***

Please explain **IN DETAIL** what the changes are in this PR and why they are needed:

- extract `user_key` in `get_from_table`
- remove duplicated parameter `key` 

## Checklist

~- [ ] I have written necessary rustdoc comments~
~- [ ] I have added necessary unit tests and integration tests~
- [x] All checks passed in `./risedev check` (or alias, `./risedev c`)

### CODE REVIEW

File: src/storage/src/hummock/validator.rs [hunk 1]
```diff
@@ -52,10 +52,13 @@ pub async fn validate_ssts(task: ValidationTask, sstable_store: SstableStoreRef)
...
             }
         };
+
+        // TODO: to use `prefetch: false` after `prefetch` be supported by
+        // SstableIteratorReadOptions
         let mut iter = SstableIterator::new(
             holder,
             sstable_store.clone(),
-            Arc::new(SstableIteratorReadOptions { prefetch: false }),
+            Arc::new(SstableIteratorReadOptions::default()),
```
Comment (hzxa21) [inline src/storage/src/hummock/validator.rs:61]: Why not keep the original implementation if we anyway want `prefetch: false`

Comment (Li0k) [reply src/storage/src/hummock/validator.rs:61]: > Why not keep the original implementation if we anyway want `prefetch: false`

Currently, this configuration does not take effect. I think it's easy to misunderstand the logic here when reading the code (for me), so I added comments and used default like any other file

File: src/storage/src/hummock/value.rs [hunk 2]
```diff
@@ -126,10 +126,10 @@ impl<'a> HummockValue<&'a [u8]> {
         }
     }
 
-    /// Copies `self` into [`HummockValue<Vec<u8>>`].
-    pub fn to_owned_value(&self) -> HummockValue<Vec<u8>> {
+    /// Copies `self` into [`HummockValue<Bytes>`].
+    pub fn to_owned_bytes_value(&self) -> HummockValue<Bytes> {
```
Comment (hzxa21) [inline src/storage/src/hummock/value.rs:130]: Bytes is design to be shared so `to_owned_bytes` seems inappropriate. Let's call it `to_bytes`.

Comment (Li0k) [reply src/storage/src/hummock/value.rs]: rename it

**Review by hzxa21 — APPROVED** [review]

**Review by xxhZs — APPROVED** [review]

**Pull request merged**